sycl/source/detail/plugin.hpp

```diff
@@ -75,6 +127,10 @@ class plugin {
     // the per_instance_user_data field.
     std::string PIFnName = PiCallInfo.getFuncName();
     uint64_t CorrelationID = pi::emitFunctionBeginTrace(PIFnName.c_str());
+    auto ArgsData =
+        packCallArguments<PiApiOffset>(std::forward<ArgsT>(Args)...);
+    uint64_t CorrelationIDWithArgs = pi::emitFunctionWithArgsBeginTrace(
```

Comment:
Do we really need a new XPTI stream for emitting arguments? Can't we just extend existing one to always print arguments?

Reply:
I think having a new stream for argument capture is a good thing - if people subscribe to the new stream, they explicitly acknowledge that this stream may have a higher cost and they would like the additional data. The older stream may be sufficient for perf tools who would like the overheads to be lower.

Reply:
I understand that, but I don't think there are currently (or ever be) users who don't want arguments because of its cost.

Reply:
Do we have an agreement here? As a compromise, we could claim the old stream as deprecated and sunset it sometime later, when all users of existing API have a chance to migrate.

Reply:
OK, let's keep 2